### PR TITLE
Pycrypto tests

### PR DESCRIPTION
**Pull request by abolla22**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -57,6 +58,8 @@ requirements:
     - python
 
 test:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:62]: Please update `test `to:
```
test:
  requires:
    - pip
    - pytest
  commands:
    - pip check
  imports:
    - Crypto
    - Crypto.Cipher
    - Crypto.Hash
    - Crypto.Protocol
    - Crypto.PublicKey
    - Crypto.Random
    - Crypto.Random.Fortuna
    - Crypto.Random.OSRNG
    - Crypto.SelfTest
    - Crypto.SelfTest.Cipher
    - Crypto.SelfTest.Hash
    - Crypto.SelfTest.Protocol
    - Crypto.SelfTest.PublicKey
    - Crypto.SelfTest.Random
    - Crypto.SelfTest.Random.Fortuna
    - Crypto.SelfTest.Random.OSRNG
    - Crypto.SelfTest.Signature
    - Crypto.SelfTest.Util
    - Crypto.Signature
    - Crypto.Util
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -39,7 +39,8 @@ source:
     - pycrypto_PR_200.diff
 
 build:
-  number: 10
+  skip: True    #[py>=38]
+  number: 11
   script:
     - set CL=/FI"%VCToolsInstallDir%INCLUDE\stdint.h" %CL%  # [win and py>=35]
     - export LDFLAGS="-headerpad_max_install_names"       # [osx]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:46]: in the host are missing packages setuptools and wheel:
```
requirements:
  build:
   ....
  host:
    - python
    - pip
    - setuptools
    - wheel
  run:
...
```

Comment (abolla22) [reply recipe/meta.yaml:46]: will update these.

File: recipe/meta.yaml (hunk 1 above)
Comment (abolla22) [reply recipe/meta.yaml:62]: the import validations are there earlier, will enhance the pip check.

**Review by skupr-anaconda — APPROVED** [review]

**Review by remkade — APPROVED** [review]
Looks good!

**Pull request merged**